epub33/reports/epub-properties-use.md

```diff
@@ -0,0 +1,604 @@
+            <th>Used By</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <th>alternate-script</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>authority</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>belongs-to-collection</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>collection-type</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>display-seq</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>file-as</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>group-position</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>identifier-type</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>meta-auth (deprecated)</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>role</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>source-of</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>term</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>title-type</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### Link Vocabulary
+#### Link Relationships
+
+The following table lists publishers who have stated that they are currently using
+the link relationships in production.
+
+<table>
+    <thead>
+        <tr>
+            <th>Property</th>
+            <th>Used By</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <th>acquire</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>alternate</th>
+            <td>
+            	<ul>
+            		<li>TBD</li>
+            	</ul>
+            </td>
+        </tr>
+        <tr>
+            <th>marc21xml-record (deprecated)</th>
```

Comment:
Same comments as before (and valid for the two entries below and to `xml-signature` and `xmp-record`): I wonder whether listing deprecated properties here makes sense.

This is also a question to @dlazin @wareid @dauwhe

Reply:
Thinking a bit further, back in the specification proper: maybe regrouping the deprecated records separately in, eg, §D3, may make the situation cleaner...

Reply:
I put them in for completeness. What I was thinking we would do is add some sort of boilerplate statement in the second column about deprecated properties not needing verification. I can try and figure something out now, but I wouldn't shift them around into a new group. That'll just make it harder to locate them. We should try to parallel the core spec as much as possible.

Reply:
What I've done for these is strike them through using `s` tags and dropped "(deprecated)" and instead added the prose:

> This property is deprecated and no longer recommended for use in EPUB Publications. It is only listed for completeness of reporting.